pkg/metrics/metrics.jsx

```diff
@@ -1420,8 +1416,9 @@ const PCPConfigDialog = ({
                                     firewalldRequest({ service: "pmproxy", title: _("Open the pmproxy service in the firewall to share metrics.") });
                                 else
                                     firewalldRequest(null);
+                                closeCallback();
                             })
-                            .catch(err => { setPending(false); setDialogError(err.toString()) });
+                            .catch(err => { closeCallback(); setPending(false); setDialogError(err.toString()) });
```

Comment:
This is wrong -- this does *not* close the dialog, but shows the error message in it. Either drop it here, or rename the attribute and the corresponding bits in the caller from "close" to something like "package-check".

Reply:
I spend a frustrating amount of time figuring out that when we do the PCP package install this runs into the `catch` with the amazing `err: "error"` (that comes from somewhere).

But sure can rename it to package-check